### ISSUE TITLE
New release candidate & merge develop into main branch?

### ISSUE THREAD

**Issue by grctest** [bitshares/beet#235]
**Is your feature request related to a problem? Please describe.**

There are several years of updates in the develop branch which have yet to be merged into the main branch.

There are several significant updates in the develop branch since release candidate 15 was released.

**Describe the solution you'd like**

Once the additional language pull request is merged, merge the current develop branch into the main branch then build a new release.

**Describe alternatives you've considered**

Direct users to the develop branch, guide them to self compile the client.

Instruct users to download built client from forked repo.

**Additional context**

Existing 3rd party beet integrations will need to update to maintain functionality.

**Comment by grctest** [comment]
@sschiessl-bcp Anything remaining in the way of merging the develop branch to main & creating a new release? Thanks

**Comment by sschiessl-bcp** [comment]
done
